var/spack/repos/builtin/packages/tensorflow-estimator/package.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class TensorflowEstimator(Package):
+    """TensorFlow Estimator is a high-level TensorFlow API that greatly
+    simplifies machine learning programming."""
+
+    homepage = "https://github.com/tensorflow/estimator"
+    url      = "https://github.com/tensorflow/estimator/archive/v1.13.0.tar.gz"
+
+    version('2.0.0-alpha', '739923c39443ec614f5348d7ff7f1d03')
+    version('1.13.0',      '50ee7a8fdd1a0abad027b63999fb464b', preferred=True)
+
+    extends('python')
+
+    depends_on('tensorflow@2.0.0-alpha0',  when='@2.0.0-alpha')
+    depends_on('tensorflow@1.13.1',        when='@1.13.0')
+
+    depends_on('bazel@0.19.0',             type='build')
+    depends_on('py-pip',                   type='build')
+    depends_on('py-funcsigs@1.0.2:',       type=('build', 'run'))
+
+    def install(self, spec, prefix):
+        tmp_path = os.path.join(env.get('SPACK_TMPDIR', '/tmp/spack'),
```

Comment:
This is going to need some explanation... I doubt `/tmp/spack` will be available on most computers.

Reply:
Should spack also check if other common locations are available: like `/cache` or `/scratch`?

Reply:
For a long time we build our software directly on a network-mounted filesystem (NFS to be precise) and bazel _really_ didn't like that so we ended up pinning this to `/tmp` which is a node-local hard drive in our cluster. As of now we are building on a dedicated build machine and deploy via singularity container to end users, so for us dropping this _should_ not make any difference. For anyone working with spack on a network-mounted filesystem, this will probably resurface the problem.